google-cloud-logging/lib/google/cloud/logging/logger.rb

```diff
@@ -54,10 +66,9 @@ class Logger
         attr_reader :labels
 
         ##
-        # A OrderedHash of Thread IDs to Stackdriver request trace ID. The
-        # Stackdriver trace ID is a shared request identifier across all
-        # Stackdriver services.
-        attr_reader :trace_ids
+        # A OrderedHash of Thread IDs to RequestInfo, keeping track of data
+        # associated with the request being handled by the current thread.
```

Comment:
Given that there is a `RequestInfo` type, I would expect a property named `request_info` to return that type, and getting an `OrderedHash` may come as a surprise. What about adding an example here and maybe also the YARD `@return` tag to help communicate the actual type?